.ambient-package-update/metadata.py

```diff
@@ -0,0 +1,44 @@
+from ambient_package_update.metadata.author import PackageAuthor
+from ambient_package_update.metadata.constants import (
+    DEV_DEPENDENCIES,
+    LICENSE_MIT,
+    SUPPORTED_DJANGO_VERSIONS,
+    SUPPORTED_PYTHON_VERSIONS,
+)
+from ambient_package_update.metadata.maintainer import PackageMaintainer
+from ambient_package_update.metadata.package import PackageMetadata
+from ambient_package_update.metadata.readme import ReadmeContent
+
+METADATA = PackageMetadata(
+    package_name="django-commons-playground",
+    github_package_group="django-commons",
+    authors=[
+        PackageAuthor(
+            name="Tim Schilling",
+            email="schillingt@better-simple.com",
+        ),
+    ],
+    maintainer=PackageMaintainer(
+        name="Tim Schilling",
+        email="schillingt@better-simple.com",
+        url="https://www.better-simple.com",
+    ),
+    company="better-simple.com",
+    license=LICENSE_MIT,
+    license_year=2024,
+    development_status="5 - Production/Stable",
+    has_migrations=False,
+    readme_content=ReadmeContent(uses_internationalisation=False),
+    main_branch="main",
+    dependencies=[
+        f"Django>={SUPPORTED_DJANGO_VERSIONS[0]}",
+    ],
```

Comment:
I feel like this shouldn't have to be specified if we're also setting `supported_django_versions=SUPPORTED_DJANGO_VERSIONS,`

Reply:
I do understand your point though this is for the pyproject toml file and it defines the "real" dependency when you install the package. This defaults to the lowest version. I added it there so it's explicit and not implicit. Hope my reasoning is understandable?

Reply:
I understand. I don't think it's clear why you're using the first element of that list. For the reader, they don't know the ordering of that list.

Reply:
That's true. I'll put this on my list of things I'll work on if we manage to make the "cooperation" work.